src/diagnostics/define_diagnostics.jl

```diff
@@ -179,130 +181,276 @@ function define_diagnostics!(land_model)
 
     ###### SoilCanopyModel ######
 
+    ## stored in p (diagnostics variables stored in the cache) ##
+
+    ## Canopy Module ##
+	
+	### Canopy - Solar Induced Fluorescence 
+	    # Solar Induced Fluorescence
     add_diagnostic_variable!(
-        short_name = "slw",
-        long_name = "Soil Liquid Water",
-        standard_name = "soil_liquid_water",
-        units = "m^3 m^-3",
-        comments = "Soil moisture, the liquid water volume per soil volume. This liquid water is located in the soil pores.",
+        short_name = "sif",
+        long_name = "Solar Induced Fluorescence",
+        standard_name = "solar_induced_fluorescence",
```

Comment:
should we say at 755 nm?

Reply:
@braghiere 

I am not sure, if future implementations could be other wavelength as well?

Reply:
Yes. This is only for 755nm. SIF can be calculate in multiple wavelengths.